client/src/Component/SignUp/SignUp.js

```diff
@@ -151,7 +151,7 @@ const SignUp = ()=>{
                     <button 
                     className="create_account_button"
                     onClick={()=>{PostData()}}>Create account</button>
-                    <p className="signup_signin_message">Already a user? <span className="signin_link">Sign in</span></p>
+                    <p className="signup_signin_message">Already a user? <a href="/signin" className="signin_link">Sign In</a></p>
```

Comment:
Replace the <a> tag with a <Link> component from react-router-dom.

Reply:
I thought of doing that but the SignIn page <a> tag was used  therefore I thought not to break the flow

Reply:
So  should i change he both pages?

Reply:
@purnima143  please confirm